ranger/config/commands.py

```diff
@@ -1743,13 +1743,26 @@ class narrow(Command):
 
     Show only the files selected right now. If no files are selected,
     disable narrowing.
+
+    To disable files when selection is active, use the :unnarrow
```

Comment:
I think you meant "filtering" or something rather than "files."

Reply:
Gah. Malapropism on my part. I reworked the docs in my new patch.